README.md

```diff
@@ -12,11 +12,6 @@ Imagine that you are the librarian of OOP University, and you need a tool to rec
 
 A basic understanding of Ruby and Git
 
-### Usage
-
-1. Install Ruby
-2. Clone this repo `https://github.com/Dagic-zewdu/school-library.git`
-
 ## Authors
 
 👤 **Dagic-zewdu**
```

Comment:
- [ ] Please let's make use of all these headers from [this Microverse README template](https://github.com/microverseinc/readme-template). Add more details in the Getting Started section(how to setup up the project locally). The License section of the Microverse README template is also missing. By adding all these details and headers, our README file will become descriptive and professional.